docs/guides/migrate-to-zeebe-user-tasks.md

```diff
@@ -0,0 +1,376 @@
+---
+id: migrate-to-zeebe-user-tasks
+title: Migrate to Zeebe user tasks
+description: "Migrating user tasks from job-based to Zeebe implementation"
+---
+
+import DocCardList from '@theme/DocCardList';
+import FormViewer from "@site/src/mdx/FormViewer";
+import YesItem from "./assets/react-components/YesItem";
+import NoItem from "./assets/react-components/NoItem";
+import TableTextSmall from "./assets/react-components/TableTextSmall";
+import userTaskMigrationDecisionHelperForm from "./assets/forms/userTaskMigrationDecisionHelperForm.js";
+import "./assets/css/cleanTable.module.css";
+import styles from "./assets/css/cleanImages.module.css";
+import APIArchitectureImg from './assets/api-architecture.png';
+import ZeebeTaskSelectionImg from './assets/zeebe-user-task-selection.png';
+
+Camunda 8.5 introduces a new [user task](/docs/components/modeler/bpmn/user-tasks/user-tasks.md) implementation type: Zeebe user tasks.
+Zeebe user tasks have a couple of benefits, including running directly on the automation engine, removing dependencies and round trips to the Tasklist web app, and a more powerful API.
+
+In this guide, you will learn
+
+- Under which circumstances you should migrate
+- To decide when to migrate and to estimate the impact on their project
+- The steps you need to take for a successful migration without interrupting your operations
+
+## Decide on your migration path
+
+Zeebe user tasks require to migrate both the user tasks in your diagrams and the task API. You can migrate in your own pace. If you should migrate now or later, and what is required to migrate, depends on your current setup and future plans.
+
+Use the following decision helper questionaire to figure out what's right for you.
+
+<FormViewer schema={ userTaskMigrationDecisionHelperForm } customStyles={{
+    border: "none",
+    padding: "0px",
+    maxWidth: "unset",
+    background: "transparent",
+    width: "calc(100% + 42px)",
+    marginLeft: "-28px"
+}} />
```

Comment:
Blocking: 

Maybe it's still in progress, but I don't see this component rendering when I run locally. I think there might be an issue with the configuration for it -- in its current state, I see nothing the whole time; when I edit the related userTaskMigrationDecisionHelperForm.js to have an empty `components: []` array, I at least see the "Loading..." placeholder flash on load. 

This is what it looks like right now when I run locally: 

<img width="760" alt="image" src="https://github.com/camunda/camunda-docs/assets/1627089/74615463-ed1b-43bb-863e-a13cb0c08800">

Reply:
Have you run npm install before? It requires an update of form-js